### PR TITLE
Unit test for #1340 fix

### PR DESCRIPTION
**Pull request by int19h**
and fixes to various minor issues caused by the change

PR for the actual fix is https://github.com/Microsoft/R-Host/pull/55

### CODE REVIEW

File: src/Debugger/Engine/Impl/AD7Property.cs [hunk 1]
```diff
@@ -313,7 +314,7 @@ public AD7Property(AD7Property parent, DebugEvaluationResult result, bool isSynt
             if (fields.HasFlag(enum_DEBUGPROP_INFO_FLAGS.DEBUGPROP_INFO_VALUE)) {
                 if (valueResult != null) {
                     // TODO: handle radix
-                    dpi.bstrValue = valueResult.GetRepresentation().Deparse;
+                    dpi.bstrValue = valueResult.GetRepresentation().Deparse.ToUnicodeQuotes();
```
Comment (int19h) [inline src/Debugger/Engine/Impl/AD7Property.cs]: This one is not directly related to the fix, but it's another place where this call was missing where it should have been present.

File: src/Host/Process [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 9f82bae74ecfdff91681bf8569dd4c49d0063397
+Subproject commit 7d4a4112debaefb566860a21981e9a562e28db7b
```
Comment (int19h) [inline src/Host/Process]: I'll update it to the actual commit once the host part goes in.

File: src/Host/Client/Test/Session/RSessionTest.Output.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+                using (var inter = await _session.BeginInteractionAsync()) {
+                    await inter.RespondAsync("'日本語'\n");
+                }
+
+                output.ToString().Should().Be("[1] \"日本語\"\n");
+            }
+        }
+    }
+}
```
Comment (huguesv) [inline src/Host/Client/Test/Session/RSessionTest.Output.cs]: Can you add an example of an inexact conversion that causes it to be escaped?

Comment (int19h) [inline src/Host/Client/Test/Session/RSessionTest.Output.cs]: It already exists in JsonTest - it started failing, that's how I noticed :) it's this data:

https://github.com/Microsoft/RTVS/blob/131fdf77f4a9786d2d55755ff5b8e83c73ede4dc/src/Debugger/Test/JsonTest.cs#L63

"δ" is lowercase delta, which got mistranslated as "d".

**Comment by huguesv** [comment]
:+1:
